gap/attr.gi

```diff
@@ -954,6 +954,50 @@ function(digraph)
   return girth;
 end);
 
+InstallMethod(DigraphOddGirth, "for a digraph",
+[IsDigraph],
+function(digraph)
+  local comps, girth, oddgirth, A, B, gr, k, comp;
+  if IsAcyclicDigraph(digraph) then
+    return infinity;
+  elif IsOddInt(DigraphGirth(digraph)) then
+    # No need to check girth isn't infinity, as we have
+    # that digraph is cyclic.
+    return DigraphGirth(digraph);
+  fi;
+  comps := DigraphStronglyConnectedComponents(digraph).comps;
+  oddgirth := infinity;
+  for comp in comps do
+    if not IsStronglyConnectedDigraph(digraph) then
+      gr    := InducedSubdigraph(digraph, comp);
```

Comment:
Weird spacing before `:=`

Reply:
Thanks James, I have made all the changes and will push them now.